### ISSUE TITLE
Staging Review finding: Work with ADE on appt calendar

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#139186]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ursa Minor
**Product:** Appointments
**Feature:** Solid Start (VASS) Scheduling

## Findings details

**VA.gov Experience Standard - issue:** [User can't easily perform the primary task free of distractions.](https://design.va.gov/about/experience-standards/#user-can-perform-the-primary-task-free-of-distractions)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @jasonday (Accessibility)

### Description

The calendar feature to pick a day and time for the call is not a great UX experience. All the radio buttons can be overwhelming, however since this is not a typical form question use case, the solution is complicated.

<details><summary>Link, screenshot or steps to recreate</summary>

![screen](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/117928-allison-4.png?raw=true)

</details>

## Recommended action

The ADE team is currently working on [solving this problem](https://github.com/department-of-veterans-affairs/va.gov-team/issues/137487). Work with ADE to find a better solution.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** [Design Foundation - Overview](https://design.va.gov/foundation/)
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by Jenny-yw** [comment]
The appointment calendar is used broadly across the Appointments tool. The team is currently addressing major accessibility issues and working on a new design. Once that’s complete, the calendar will be updated throughout the tool, including here. (see [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/113652))
